iphone/Classes/TiMediaVideoPlayerProxy.m

```diff
@@ -117,8 +117,10 @@ - (void)addNotificationObserver
 
 - (void)removeNotificationObserver
 {
+  if ([self observationInfo]) {
+    [self removeObserver:self forKeyPath:@"url"];
```

Comment:
Does this only apply for the url KVO or possibly others as well (once triggered)?

Reply:
For others also we can apply. But 'movie' object already becomes nil. So no need to apply this check for others.